app/src/main/java/ca/marshallasch/veil/PostListAdapter.java

```diff
@@ -33,15 +29,15 @@
     /**
      * This is the cell for each post in the list.
      */
-    public static class ViewHolder extends RecyclerView.ViewHolder {
+     static class ViewHolder extends RecyclerView.ViewHolder {
```

Comment:
Default is private and makes sense since it only used in this class. But we still should add the private keyword just to improve readability.

Reply:
default is package-private not private, it does not work if it is private. since it is used in the parent class